src/views/components/survey_components/survey_list.vue

```diff
@@ -0,0 +1,56 @@
+<!-- information that I need in this file -->
+<!-- information that I will send in this file -->
+<style>
+li {
+    display: block;
+    font-size: 1.2em;
+    margin: 0.4em;
+    cursor: pointer;
+}
+</style>
+
+<template>
+    <div>
+        <h1>Surveys avaliable</h1>
+        <ul>
+            <li v-for="survey in surveys" @click="survey_to_question(survey)"> Survey ID: {{survey.survey_id}} &ensp; Survey name: {{survey.name}}</li>
+        </ul>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "survey_list",
+    data: function () {
+        return {
+            surveys: [
+                {
+                    "survey_id": 0,
+                    "name": "best survey0"
+                },
+                {
+                    "survey_id": 1,
+                    "name": "best survey1"
+                },
+                {
+                    "survey_id": 2,
+                    "name": "best survey2"
+                },
+            ]
+        }
+    },
+    methods: {
+        // this function will direct the user to a specific question page depending on which link
+        // the user is clicking.
+        survey_to_question: function (survey) {
+            window.location.href = "/#/surveys/question_time?survey_id=" + survey.survey_id +
```

Comment:
there is a function called router.push. It pushes parameters and values onto the url. This function also saves the current url in the history. If you don't want the current url to be in the history, you can use router.replace.

Reference the navbar component as an example.